README.md

```diff
@@ -6,8 +6,26 @@
 [![Releases](https://img.shields.io/github/release-pre/cloudevents/sdk-go.svg)](https://github.com/cloudevents/sdk-go/releases)
 [![LICENSE](https://img.shields.io/github/license/cloudevents/sdk-go.svg)](https://github.com/cloudevents/sdk-go/blob/master/LICENSE)
 
-**NOTE: This SDK is still considered work in progress, things might (and will)
-break with every update.**
+**NOTE: This SDK is still considered work in progress.**
+
+**With v1.0.0:**
+
+The API that exists under [`pkg/cloudevents`](./pkg/cloudevents) will follow
+semver rules. This applies to the root [`./alias.go`](./alias.go) file as well.
+
+Even though `pkg/cloudevents` is v1.0.0, there could still be minor bugs and
+performance issues. We will continue to track and fix these issues as they come
+up. Please file a pull request or issue if you experiance problems.
+
+The API that exists under [`pkg/bindings`](./pkg/bindings) is a new API that
+will become SDK v2.x, and will replace `pkg/cloudevents`. This area is still
```

Comment:
I think we should not replace `pkg/cloudevents` with `pkg/bindings`, because the `cloudevent.Client` API is simple, yet powerful for a lot of use-cases. IMO we should say that we plan to reimplement/replace `pkg/cloudevents/transport` using `pkg/bindings`

Reply:
ah yeah, the `pkg/cloudevents/client` will get pushed out to `pkg/client`

Reply:
Those details will get filled into the v2 plan.